### DISCUSSION TITLE
translate to Chinese

### DISCUSSION THREAD

**Discussion by OS-Lihua** [Cyfrin/foundry-full-course-cu#2753] [Q&A] [answered]
I want to translate some pages into Chinese, so that more Chinese developers can learn more easily, if I submit the translated results to the main branch, can it be merged?

**Comment by EngrPips** [comment, marked as answer]
Hello @OS-Lihua, I believe you have figured this out already. Right?

**Reply by OS-Lihua** [reply]
yeah

**Reply by EngrPips** [reply]
Awesome. Keep crushing it, Friend.
